openreview/venue/process/paper_recruitment_pre_process.py

```diff
@@ -17,6 +18,15 @@ def process(client, edit, invitation):
 
     submission = client.get_notes(note.content['submission_id']['value'])[0]
 
+    invite_edges = client.get_edges(invitation=invite_assignment_invitation, head=submission.id, tail=user)
+    if not invite_edges:
+        profile = openreview.tools.get_profile(client, user)
+        if profile:
+            invite_edges = client.get_edges(invitation=invite_assignment_invitation, head=submission.id, tail=profile.id)
```

Comment:
why do you search the invitation edge by profile?

Reply:
From the issue: 
> We can get the user and the submission id from the url. If no edge found for that user, we can try to search the edge by profile.id. Sometimes there is a profile merge and edges are renamed.